lib/models/settings/settings.dart

```diff
@@ -23,9 +23,9 @@ abstract class Settings implements Built<Settings, SettingsBuilder> {
   static SettingsBuilder initBuilder() {
     return SettingsBuilder()
       ..darkTheme.brightness = ThemeBrightness.dark
-      ..darkTheme.colors = ThemeColors.crowdleague.toBuilder()
```

Comment:
changed so the app is in greyscale mode to start with, should have always been this way

Reply:
nice pickup, make those users focus on what matters!